test/utilities/connectivity_testing.go

```diff
@@ -87,6 +96,42 @@ func CreateNetworkConf(cniVersion string, name string, pluginType string,
 		DNS:            *dns,
 		AdditionalArgs: addArgs,
 	}
+	
+	return &netConf
+}
+
+func CreateDualStackNetworkConf(cniVersion string, name string, pluginType string,
```

Comment:
kind of hard to find a pleasant way to style these long function signatures, but I found [this online discussion on the topic](https://www.reddit.com/r/golang/comments/7e8dl0/how_do_you_or_your_team_format_long_function/). in my opinion putting each parameter on a new line or declaring a struct for the parameters may look nicer, but your call

Reply:
I updated using one parameter per line convention.